src/_posts/introducing-aframe.md

```diff
@@ -6,7 +6,7 @@ layout: blog
 
 ### Today the MozVR team releases the first version of A-Frame: an open source framework for creating WebVR experiences with markup.
 
-A-Frame makes it easy for web developers to create virtual reality experiences that work across desktop, iPhone (Android support coming soon), and the Oculus Rift.
+A-Frame makes it easy for web developers to create virtual reality experiences that work across desktop, iPhone, Android, and the Oculus Rift.
```

Comment:
yeah, I thought about that but "desktop" is not an operating system (nor is a platform; it's a form factor).

this is all phrased awkwardly because Oculus Rift is for desktop only. I know Josh meant to say "2D desktop" and "VR desktop w/ Oculus Rift."